docs/biovalidator_benchmarks/2023.01.23_benchmarks/local_endpoint/README.md

```diff
@@ -0,0 +1,87 @@
+# JSON validation summary
+The script [``create_benchmarks.py``](https://github.com/EbiEga/ega-metadata-schema/blob/main/.github/scripts/create_benchmarks.py) validated all JSON files in a directory against a validation server endpoint, recording valuable parameters of its execution.
+Overall execution parameters:
+| Date                | Passed validation   |   Validation time (s) | Input directory                  | Used endpoint                  |   Nº of executions |   Nº of files |   Nº of properties |   Nº of ontology validations |
+|:--------------------|:--------------------|----------------------:|:---------------------------------|:-------------------------------|-------------------:|--------------:|-------------------:|-----------------------------:|
+| 2023-01-23 18:42:48 | True                |               1108.75 | 2023.01.23_benchmarks/json_docs/ | http://localhost:3020/validate |                250 |            67 |           21753000 |                       999000 |
+
+All files generated in this execution are stored in this directory (``2023.01.23_benchmarks/local_endpoint_1``) with this same README. These files are:
+1. Two CSV files: the complete set of validation parameters for each file and validation attempt ([``2023.01.23_benchmarks/local_endpoint/complete_df.csv``](summary_df.csv)) and a summary of those ([``2023.01.23_benchmarks/local_endpoint/summary_df.csv``](summary_df.csv)), also displayed below.
+2. Three graphs (all inserted below) of the full set of parameters: a 3D graph ([``2023.01.23_benchmarks/local_endpoint_1/3D_scatterPlot.png``](3D_scatterPlot.png)) of the validation time, number of properties, and number of ontology validations; two 2D scatter subplots ([``2023.01.23_benchmarks/local_endpoint_1/2D_scatterPlot.png``](2D_scatterPlot.png)) of the validation time versus the other two variables; and a 2D density plot ([``2023.01.23_benchmarks/local_endpoint_1/2D_densityPlots.png``](2D_densityPlots.png)) containing 2 subplots: one for each variable versus the validation time
+
+Graphical representations:
+
+![3D Scatter plot](3D_scatterPlot.png)
+![2D Scatter plots](2D_scatterPlot.png)
+![2D Density plot](2D_densityPlots.png)
```

Comment:
Plots are cool! Looks like the validation scales linearly with the number of attributes and ontologies to validate - this is useful knowledge :)

Reply:
Glad to hear that :) 
Do bear in mind the co-dependency between the number of properties and number of ontology validations (observable in the ``X`` vs ``Y`` axes), which depends on the set of objects we took for the benchmarks. In our case I did a random picking of our examples, which leads to a uniform distribution of properties vs ontology validations.

We could also test benchmarks with many properties and no validations, and the other way around, to see which one is harder for Biovalidator to process.